src/main/java/org/nest/symboltable/predefined/PredefinedFunctions.java

```diff
@@ -147,13 +147,13 @@
 
     final MethodSymbol i_sum = createFunctionSymbol(CURR_SUM);
     i_sum.addParameterType(getType("pA"));
-    i_sum.addParameterType(getBufferType());
+    i_sum.addParameterType(getRealType());
```

Comment:
We check this with an extra different coco. It must be not only `BufferType`, but also a special type of the expression.